Makefile

```diff
@@ -244,7 +244,7 @@ catalog-push: ## Push a catalog image.
 
 .PHONY: authenticator-build
 authenticator-build: ## Build the authenticator image.
-	docker build -f ./images/eapol-authenticator/Dockerfile -t $(AUTHENTICATOR_IMG) ./images/eapol-authenticator/
```

Comment:
I don't see a removal of this older Dockerfile in this PR - Should we do that too?